### PR TITLE
Minor improving the unit-tests

### PR DESCRIPTION
**Pull request by WestleyK**
This PR improves the unit tests, while running them, test containers are ignored and not modified. And combine the tests that require to be ran in order.

### CODE REVIEW

File: pkg/sif/create_test.go [hunk 1]
```diff
@@ -69,29 +70,29 @@ func TestCreateContainer(t *testing.T) {
 
 	// test container creation without any input descriptors
 	if _, err := CreateContainer(cinfo); err != nil {
-		t.Error("CreateContainer(cinfo): should allow empty input descriptor list")
+		t.Errorf("CreateContainer(cinfo): should allow empty input descriptor list")
```
Comment (jscook2345) [inline pkg/sif/create_test.go]: This doesn't need to be `Errorf` unless a format string is used. Did you mean to include one?

File: pkg/sif/create_test.go [hunk 2]
```diff
@@ -158,48 +157,81 @@ func TestAddObject(t *testing.T) {
 	// open up the data object file for this descriptor
 	partHandle, err := os.Open(parinput.Fname)
 	if err != nil {
-		t.Error("CreateContainer(cinfo): read data object file:", err)
+		t.Errorf("os.Open(parinput.Fname): unable read data object file: %s", err)
```
Comment (jscook2345) [inline pkg/sif/create_test.go]: `unable to read`

File: pkg/sif/create_test.go [hunk 3]
```diff
@@ -345,3 +354,22 @@ func TestSetPrimPart(t *testing.T) {
 		}
 	}
 }
+
```
Comment (jscook2345) [inline pkg/sif/create_test.go]: this should be in a common package somewhere for reuse I think

File: pkg/sif/create_test.go (hunk 1 above)
Comment (mem) [reply pkg/sif/create_test.go]:
```suggestion
		t.Error("CreateContainer(cinfo): should allow empty input descriptor list")
```

File: pkg/sif/create_test.go (hunk 2 above)
Comment (mem) [reply pkg/sif/create_test.go]:
```suggestion
		t.Errorf("os.Open(parinput.Fname): unable to read data object file: %s", err)
```

File: pkg/sif/create_test.go (hunk 3 above)
Comment (WestleyK) [reply pkg/sif/create_test.go]: Suggestions?

**Pull request merged**